cmd/kube-mgmt/main.go

```diff
@@ -94,6 +95,7 @@ func main() {
 	rootCmd.Flags().VarP(&params.replicateNamespace, "replicate", "", "replicate namespace-level resources")
 	rootCmd.Flags().VarP(&params.replicateCluster, "replicate-cluster", "", "replicate cluster-level resources")
 	rootCmd.Flags().StringVarP(&params.replicatePath, "replicate-path", "", "kubernetes", "set path to replicate data into")
+	rootCmd.Flags().StringSliceVarP(&params.replicateIgnoreNs, "ignore-namespaces", "", []string{"opa"}, "namespaces that are ignored by replication")
```

Comment:
Use ”replicate-ignore-namespaces” to avoid ambiguity

Reply:
fixed, thanks for the review.